src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    function, normally used for debug purposes. `fiber.top()` shows all alive
+    fibers and their CPU consumption.
+]])
+
+M['fiber.top.enabled'] = format_text([[
+    Enable or disable the `fiber.top()` function.
+
+    Enabling `fiber.top()` slows down fiber switching by about 15%, so it is
+    disabled by default.
+]])
+
+M['fiber.worker_pool_threads'] = format_text([[
+    The maximum number of threads to use during execution of certain
+    internal processes (for example, `socket.getaddrinfo()` and `coio_call()`).
+]])
+
+-- }}} fiber configuration
+
+-- {{{ flightrec configuration
+
+M['flightrec'] = format_text([[
+    The flightrec section describes options related to the flight recorder
+    configuration.
+]])
+
+M['flightrec.enabled'] = 'Enable the flight recorder.'
+
+M['flightrec.logs_log_level'] = format_text([[
+    Specify the level of detail the log has. The default value is 6
+    (`VERBOSE`). You can learn more about log levels from the log_level
+    option description. Note that the `flightrec.logs_log_level` value might
+    differ from `log_level`.
+]])
+
+M['flightrec.logs_max_msg_size'] = format_text([[
+    Specify the maximum size (in bytes) of the log message. The log message
+    is truncated if its size exceeds this limit.
+]])
+
+M['flightrec.logs_size'] = format_text([[
+    Specify the size (in bytes) of the log storage. You can set this option
+    to 0 to disable the log storage.
+]])
+
+M['flightrec.metrics_interval'] = format_text([[
+    Specify the time interval (in seconds) that defines the frequency
+    of dumping metrics. This value shouldn't exceed `flightrec.metrics_period`.
+]])
+
+M['flightrec.metrics_period'] = format_text([[
+    Specify the time period (in seconds) that defines how long metrics
+    are stored from the moment of dump. So, this value defines how much
+    historical metrics data is collected up to the moment of crash. The
+    frequency of metric dumps is defined by `flightrec.metrics_interval`.
+]])
+
+M['flightrec.requests_max_req_size'] = format_text([[
+    Specify the maximum size (in bytes) of a request entry.
+    A request entry is truncated if this size is exceeded.
+]])
+
+M['flightrec.requests_max_res_size'] = format_text([[
+    Specify the maximum size (in bytes) of a response entry.
+    A response entry is truncated if this size is exceeded.
+]])
+
+M['flightrec.requests_size'] = format_text([[
+    Specify the size (in bytes) of storage for the request and
+    response data. You can set this parameter to 0 to disable
+    a storage of requests and responses.
+]])
+
+-- }}} flightrec configuration
+
+-- {{{ iproto configuration
+
+M['iproto'] = format_text([[
+    The iproto section is used to configure parameters related to communicating
+    to and between cluster instances.
+]])
+
+-- {{{ iproto.advertise configuration
+
+M['iproto.advertise'] = format_text([[
+    URIs for clients to let them know where to connect.
+]])
```

Comment:
I would say 'for cluster members and external clients', because there is `iproto.advertise.client` purely about the (external) clients.

Reply:
Ok. Let's do it this way.